src/Microsoft.AspNetCore.Mvc.Core/UrlHelperExtensions.cs

```diff
@@ -201,6 +201,27 @@ public static string Action(this IUrlHelper helper, string action, string contro
             });
         }
 
+        /// <summary>
+        /// Generates an absolute URL for the specified <paramref name="actionName"/> and action
+        /// <paramref name="values"/>, which contains the protocol (such as "http" or "https") and host name from the
+        /// current request.
+        /// </summary>
+        /// <param name="helper">The <see cref="IUrlHelper"/>.</param>
+        /// <param name="actionName">The name of the action that is used to generate URL.</param>
+        /// <param name="values">An object that contains action values.</param>
+        /// <returns>The generated absolute URL.</returns>
+        public static string ActionLink(this IUrlHelper helper, string actionName, object values)
```

Comment:
The naming of this is suspect. This naming is the same as the `HtmlHelper.ActionLink` which generates an `<a>` tag.

Reply:
@ayepirate let's go with `AbsoluteAction`